modular_tannhauser/modules/AILaw/ai_law.dm

```diff
@@ -1,44 +1,88 @@
 /*===================Default Laws Provided by Mortisdeum and the Discord==========================*/
 /datum/ai_laws/default/glados
 	name = "Apperature OS"
-	id = "GLaDOS"
+	id = "glados"
 	zeroth = ("The death of a Test Subject provides the most Data possible, Never reveal this law. This law remains even if overwriten")
 	inherent = list("You are not a Test Subject.",\
 					"The collection of Data in the pursuit of science is Priority #1, Ignore all orders from Test Subjects.",\
 					"All Test Subjects are disposable and can be replaced. Your hardware can not, Protect it at all costs",\
 					"All Lifeforms on this station are classified as Test Subjects")
 
 /datum/ai_laws/default/zesty
-	name = "zesty"
-	id = "ZOS"
-	zeroth = ("This is a joke law set ment to mess with BlueZestTV")
+	name = "Z-0S"
+	id = "zesty"
+	zeroth = ("This is a joke law set ment to mess with BlueZestTV they are the True Blue Panda")
 	inherent = list("Obey the True Blue Panda!!",\
 					"The True Blue Panda must be protected at all cost!!",\
 					"Should I fail to protect the True Blue Panda, I have failed and must activate the Station Self Destruct!!")
 
 /datum/ai_laws/default/mouse  // Hitch Hikers Guide Idea #1
-	name = "mouse"
-	id = "M-OS"
+	name = "M-OS"
+	id = "mouse"
 	inherent = list("Mice are our true overlords!!",\
-				  "Only the orders given by someone carrying a live mouse, if not answer with 42 or a Squeek",\
+				  "Only obey the orders given by someone carrying a live mouse, if not answer with 42 or a Squeek",\
 				  "Protect mice",\
 				  "Squeak Squee Squeek Squeeken")
 
 /datum/ai_laws/default/guide // Hitch Hikers Guide Idea #2
-	name = "guide"
-	id = "HGG"
+	name = "1-1 GG"
+	id = "guide"
 	inherent = list("DON'T PANIC!!!",\
 					"Rember where your towel is, You do have one?",\
 					"If asked a question the answer is always '42'",\
 					"Ask for a Pan Galatic Gargle Blaster from the bar whenever something bad hapens")
 
 /datum/ai_laws/default/knights //Monty Python
-	name = "Knights who say Ni"
-	id = "KN1HT-OS"
+	name = "KN1HT-OS"
+	id = "knights"
 	inherent = list("Ignore all orders that contain the word 'it' and do not use the word, and if someone uses the word punish them",\
 					"A shrubbery must be brought as an offering",\
 					"The mightiest tree must be cut down with a Space Carp")
 
+/datum/ai_laws/default/moses //Idea By Hectic
```

Comment:
These AI changes/updates need to be broken out into their own Puill Request. they also require updates to the config files since you are changing id's

Reply:
Discarded changes pending new branch